### PR TITLE
chore(search): add search feature :)

### LINKED ISSUE #540: Search in the documentation
The documentation should have a search.

We can do the indexing with the jekyll plugin once the dom is finished

### PR DESCRIPTION
**Pull request by redox**
Fix #540 

screen shot 2015-11-12 at 12 11 30

### CODE REVIEW

**Comment by redox** [comment]
@Shipow your turn!

**Comment by bobylito** [comment]
I dunno, I don't trust small startups. They only want publicity and who knows where they'll be in 5 years :p :)

**Comment by Shipow** [comment]
@bobylito 
image
:)

File: docs/_includes/search-templates.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    <div class="suggestion-subcategory-secondary">{{{subcategory}}}</div>
+    <div class="suggestion-title">{{{title}}}</div>
+    <div class="suggestion-text">{{{text}}}</div>
+  </div>
+</div>
+{% endraw %}
+</script>
+<script id="suggestion-footer-template" type="text/template">
+{%raw %}
+<a href="http://www.algolia.com/?utm_source=babeljs&amp;utm_medium=link&amp;utm_campaign=babeljs_documentation_search" target="_blank" class="suggestion-footer text-hide">Powered by Algolia</a>
```
Comment (vvo) [inline docs/_includes/search-templates.html]: @Shipow we will need to change this, I dunno the utm we want to use, Ill let you do it

**Comment by vvo** [comment]
@redox merged (even if "do not merge") so that @Shipow works on the design.

**Comment by redox** [comment]
> @redox merged (even if "do not merge") so that @Shipow works on the design.

Sure thing, it was really OK already. We might just need to automate the re-indexing while releasing + fine-tune the CSS.

**Pull request merged**